roles/beaconchain_explorer_aio/defaults/main.yml

```diff
@@ -10,7 +10,7 @@ beaconchain_explorer_aio_sql_init_file_url: https://raw.githubusercontent.com/go
 beaconchain_explorer_aio_docker_compose_file: templates/docker-compose.yml.j2
 
 beaconchain_explorer_aio_installation_dir: /opt/beaconchain-explorer-aio
-
+beaconchain_explorer_aio_network: []
```

Comment:
Maybe rename to `beaconchain_explorer_aio_docker_networks` <-- Notice _docker_ + plural.